src/js/player.js

```diff
@@ -2931,6 +2931,45 @@ class Player extends Component {
     if (isEvented(this)) {
       this.trigger('playerreset');
     }
+    this.resetControlBarUI();
```

Comment:
I wonder if this should come before the event being triggered above? It seems like we'd want to make our changes before we notified that we had reset the player.

Reply:
It does makes sense. However, since the resetting of the control bar UI itself triggers certain tech events, couldn't the player reset immediately after cause, in certain conditions, a change of behavior in how the triggered events are handled?

Reply:
@misteroneill that makes perfect sense. I have now moved it to before the event is triggered.